plugin.video.frikanalen/addon.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2016 Petter Reinholdtsen
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+import routing
+import xbmcplugin
+from xbmcgui import ListItem
+from xbmcplugin import addDirectoryItem
+from xbmcplugin import addDirectoryItems
+from xbmcplugin import endOfDirectory
+
+import frikanalen
+
+plugin = routing.Plugin()
+
+@plugin.route('/')
+def root():
+    items = [
+        (plugin.url_for(live), ListItem("Direkte"), True),
```

Comment:
would be nice if you could turn those listitems labels into localized strings

Reply:
I would love to make them translatable, but have not been able to figure out how yet.

Reply:
See https://kodi.wiki/view/Language_support